content/post/stats_at_work.md

```diff
@@ -0,0 +1,95 @@
+MA: Oh cool - so there's a link between p-value and effect size.
+
+DS: Yup. And the sample size.  
+MA: Right that was the next thing I was curious about. How did you know what the right sample size was?
+
+DS: Yeah, so that's all about figuring out how many users we needed to be confident in our conclusion.  
+MA: Is that the confidence interval?
+
+DS: Sort of. It's related. The confidence interval is more about telling us the interval around our estimate. Remember, we're estimating the difference between A's conversion rate and B's conversion rate.  
+MA: How does sample size fit in? Is there a formula for that?
+
+DS: It's all one formula actually! Given a significance threshold, an effect size and a power level, I can calcu-  
+MA: Wait what's power?
+
+DS: Oh sorry. Power is a term that's often overlooked because it's usually baked into the calculations. Power is sometimes used in place of sample size because they are very much linked, even though they are different concepts. Let's take a step back. Recall there are two things we want to check for in this A/B test. First, that A is truly bigger than B by at least the amount we care about (the effect size). Second, we want to be confident in our conclusion.  
+MA: And that we have a big enough sample size. 
+
+DS: Yeah - confidence in our estimate and sample size are effectively the same thing. Power is just another way to measure the confidence in our p-value calculation. When I have a small sample, the p-value that I calculate might not be very reliable. You can imagine that the error bars around my p-value estimate are very wide.  
+MA: Kind of like how a confidence interval around our estimate can be wide if our sample is small? I'm assuming if the sample is large our confident interval is narrow? 
+
+DS: Precisely.  
+MA: Cool. Power and sample size and confidence in our estimate are all related. But what sample size are we aiming for? Like what's the "5% significance level" equivalent for sample size?
+
+DS: Well, that varies. You're right, we set our significance threshold to 5%. We also set our desired effect size to 0.5%. Now the question is - how many data points do we need to be confident that our conclusion can be trusted. But let me ask you - how do we quantify "trust level"?  
+MA: I don't know. 95%?
+
+DS: Yeah that's tempting cause we chose 95% for the confidence level (which results in a 5% significance threshold level) to determine if the p-value is significant or not. Let's flip the "trust level" question. Let's consider what our trust level would be with 10 users.  
+MA: Hmm...probably small.
+
+DS: Right - small sample, small trust level. Well actually it's called power. Small sample, low power. High sample, high power. Sometimes you may read that a test with a large sample size has a lot of power to detect differences when they actually exist.  
```

Comment:
Same as comment above -- this doesn't align with my understanding of power, p-values, and sample-size

Reply:
"trust level" is, again, our trust in the ability to detect a true difference. I can clarify.